wfe/web-front-end.go

```diff
@@ -340,7 +340,7 @@ func (wfe *WebFrontEndImpl) NewRegistration(response http.ResponseWriter, reques
 
 	if existingReg, err := wfe.SA.GetRegistrationByKey(*key); err == nil {
 		logEvent.Error = "Registration key is already in use"
-		response.Header().Set("Location", fmt.Sprintf("%s%d", wfe.RegBase, existingReg.ID))
+		response.Header().Set("Location", fmt.Sprintf("%s%v", wfe.RegBase, existingReg.ID))
```

Comment:
Yeah, it was a mistake to have that patch commit end up in this PR; clearly was moving fast on the last rebase.